test/UnitTests/PlatformServices.Universal.Unit.Tests/Services/UniversalFileOperationsTests.cs

```diff
@@ -117,6 +117,57 @@ public void GetNavigationDataShouldReturnNullFileName()
             Assert.AreEqual(-1, minLineNumber);
         }
 
+        // Enable these tests when we take dependency on TpV2 object model
+        // In Tpv1 UWP Object model these below methods are not defined.
```

Comment:
Would we ever do this given that we want to support UWP in 14.0?

Reply:
If not, how do we run these tests?

Reply:
we would move to Tpv2 object model, once we have made it default. Right now I do not want to go into problem of acquiring it.

Reply:
hmm.. This is a debt we are carrying forward. Please file a bug to track this.